apps/emqx_auth_mongo/test/emqx_auth_mongo_SUITE.erl

```diff
@@ -686,7 +686,7 @@ heal_failure(FailureType, ProxyHost, ProxyPort) ->
     end.
 
 switch_proxy(Switch, ProxyHost, ProxyPort) ->
-    Url = "http://" ++ ProxyHost ++ ":" ++ integer_to_list(ProxyPort) ++ "/proxies/mongo",
+    Url = "http://" ++ ProxyHost ++ ":" ++ integer_to_list(ProxyPort) ++ "/proxies/mongo_single",
```

Comment:
This can't be changed here in CE because it doesn't match the current [`toxiproxy.json`](https://github.com/zmstone/emqx/blob/893aadb826e9adf17c375fb9bf370cca413118a7/.ci/docker-compose-file/toxiproxy.json) and the [mongo container names](https://github.com/zmstone/emqx/blob/893aadb826e9adf17c375fb9bf370cca413118a7/.ci/docker-compose-file/docker-compose-mongo-tcp.yaml#L4-L5) currently being defined.

Maybe drop this diff?